app/services/facets/tagging_update_publisher.rb

```diff
@@ -69,21 +69,32 @@ def generate_links_payload
       facet_groups_content_ids = fetch_content_ids(:facet_groups)
       facet_values_content_ids = fetch_content_ids(:facet_values)
       finder_content_ids = [FinderService::LINKED_FINDER_CONTENT_ID]
+      ordered_related_items_ids = ordered_related_links
 
       if facet_values_content_ids.any?
         facet_groups_content_ids.push(facet_group_content_id)
+        ordered_related_items_ids.unshift(FinderService::LINKED_FINDER_CONTENT_ID)
       else
         facet_groups_content_ids.delete(facet_group_content_id)
         finder_content_ids = []
+        ordered_related_items_ids.delete(FinderService::LINKED_FINDER_CONTENT_ID)
```

Comment:
I'm a huge fan of the simplicity here, huge
I _think_ it's clear enough to a casual viewer that we want to remove the finder content id if the facet_value is missing, I was going to suggest putting something in the commit message but you already have! What a pro 🏅